README.md

```diff
@@ -90,22 +105,35 @@ Liquidity: 21972098821216706277
 Tick:      205282
 Price:     "1.216568804789000000000000000000000000000e+3"
 ```
+
 You may also build the executable with `cargo build`, which will output a binary in `target/`
 
+## Configuration File
+
+To run arbiter with a configuration file, we can run the command `cargo run see --config true` or, if we have built the latest changes, `arbiter see --config true`. This will load in configuration settings from a `config.toml` file located in `./crates/cli/src/`. Feel free to make custom configuration files for your needs. The logic that finds the file is in `crates/cli/src/config/mod.rs`; remember to add the path of your new file to the list of file paths.
```

Comment:
We need to update this so it reflects the latest changes of being able to run `arbiter see --config`, without the true parameter.